docs/source/using_the_ve/getting_started.md

```diff
@@ -20,7 +20,7 @@ language_info:
   name: python
   nbconvert_exporter: python
   pygments_lexer: ipython3
-  version: 3.11.9
+  version: 3.12.0rc3
 ---
 
 # Getting started
```

Comment:
At the first mention of `pip` or a command, should we assume less coding experience of the users and say explicitly where to run it? For example, before `pip install` below, say something like "...open up the Terminal and type:"